.circleci/config.yml

```diff
@@ -252,8 +252,8 @@ commands:
           command: |
             mkdir ~/.aws
             echo [default] >> ~/.aws/credentials
-            echo aws_access_key_id = $AWS_ACCESS_KEY_ID >> ~/.aws/credentials
-            echo aws_secret_access_key = $AWS_SECRET_ACCESS_KEY >> ~/.aws/credentials
+            echo aws_access_key_id = $SHAPESHIFT_AWS_ACCESS_KEY_ID >> ~/.aws/credentials
```

Comment:
I had considered this, but decided against bringing the `organization` parameter into this change. Since we introduce that parameter in the `taxi-stake` branch, this change will be wiped out as soon as that branch is merged. Thus I opted for hard-coding SHAPESHIFT.

Reply:
Ok, that makes sense!